src/lib/selectors.ts

```diff
@@ -382,4 +384,17 @@ const selectors: Record<number, { readonly selector: bigint; readonly name?: str
   },
   // end:generate
 }
+
+export const aptosSelectors: Selectors = {
```

Comment:
Please, add the `generate:` comments to generate from [here](https://github.com/smartcontractkit/chain-selectors/blob/main/selectors_aptos.yml), so we make sure these are kept up-to-date